claim.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: vshn.appcat.vshn.io/v1
+kind: VSHNPostgreSQL
+metadata:
+  name: widera-test
+spec:
+  parameters:
+    service:
+      majorVersion: "15" 
+      pgSettings:
+        timezone: Europe/Zurich 
+    backup: 
+      schedule: "0 23 * * *"
+      retention: 12
+    maintenance:
+      dayOfWeek: wednesday
+      timeOfDay: '23:30:00'
+    monitoring:
+      email: "lukasz.widera@vshn.ch"
```

Comment:
Are you sure you want your mail address out there? We get enough spam as it is :)